lms/views/error.py

```diff
@@ -36,23 +35,6 @@ def http_error(exc, request):
     return {"message": str(exc)}
 
 
-def lti_launch_error(exc, request):
-    """
-    Handle an invalid LTI launch request.
-
-    Code raises :exc:`lms.exceptions.LTILaunchError` if there's a problem
-    with an LTI launch request, such as a required LTI launch parameter
-    missing. When this happens we:
-
-    1. Report the error to Sentry.
-    2. Set the HTTP response status code to 400 Bad Request.
-    3. Show the user an error page containing the specific error message
-    """
-    sentry_sdk.capture_exception(exc)
-    request.response.status_int = 400
```

Comment:
The `http_error` view gets the status code from the context (`exc`) instead of always using 400. Will this affect the status code of the response in any scenarios?